provider/packet/environ.go

```diff
@@ -171,7 +170,7 @@ func (e *environ) Destroy(ctx context.ProviderCallContext) error {
 }
 
 func (e *environ) DestroyController(ctx context.ProviderCallContext, controllerUUID string) error {
-	insts, err := e.getPacketInstancesByTag(map[string]string{"juju-is-controller": "true", "juju-controller-uuid": controllerUUID})
+	insts, err := e.getPacketInstancesByTag(map[string]string{"juju-controller-uuid": controllerUUID})
```

Comment:
Also, try to avoid hardcoding the tag names. You can use the constants in https://github.com/juju/juju/blob/develop/environs/tags/tags.go